src/components/dashboard/rooms-tab.yml

```diff
@@ -1,28 +1,13 @@
 roomsTab:
   info:
-    en: Here you find all the rooms you have created and the rooms that you are a member of.
-    de: Hier findest du alle Räume, die du erstellt hast und in denen du Mitglied bist.
-  joined:
-    en: Joined
-    de: Eingetreten
-  role:
-    en: Role
-    de: Rolle
-  member:
-    en: Member
-    de: Mitglied
-  sortedByOwner:
-    en: Sorted by owner
-    de: Nach Eigentümer sortiert
-  sortedByRole:
-    en: Sorted by role
-    de: Nach Rolle sortiert
+    en: Here you find all the rooms you have created and the rooms that were invited to.
+    de: Hier findest du alle Räume, die du erstellt hast und in denen du eingeladen wurdest.
```

Comment:
.. und in die du eingeladen wurdest.

Reply:
Thanks for correcting (and explaining to me). Changed.